packages/botonic-plugin-flow-builder/src/action.tsx

```diff
@@ -46,26 +46,28 @@ export class FlowBuilderAction extends React.Component<FlowBuilderActionProps> {
       locale
     )
 
+    if (flowBuilderPlugin.trackEvent) {
+      // TODO: track all targets nodes?
```

Comment:
@asastre should we track just the "parent" or all the follow-up/target contents?

Reply:
cool! let's iterate it for next sprints